.github/workflows/build.yaml

```diff
@@ -0,0 +1,52 @@
+name: Build and Upload
+
+on:
+    [push]
+permissions:
+    contents: write
+  
+jobs:
+    build-and-push:
+        # container:
+        #   image: ubuntu:22.04
+        #   options: --privileged
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                image: [
+                        mirteopi, 
```

Comment:
By default we will only release opizero2, and raspberry pi. We should not by default include orangepizero1. And in doubt about 3b.

Reply:
I've removed zero1, changed the naming and currently testing rpi4b

Reply:
Added rpi4b, unable to test it